.yarnrc.yml.dist

```diff
@@ -0,0 +1,12 @@
+defaultSemverRangePrefix: ""
+nodeLinker: node-modules
+npmScopes:
+  fortawesome:
+    npmRegistryServer: "https://npm.fontawesome.com/"
+    npmAlwaysAuth: true
+    npmAuthToken: "FILL-ME"
```

Comment:
Need to update https://github.com/bethinkpl/design-system/blob/b09a273d112dbe529993a35c6c0dbeeb9e06e067/.github/workflows/ci.yml#L18 too.